src/apiCalls.js

```diff
@@ -0,0 +1,23 @@
+// import axios from 'axios'
```

Comment:
The team wants to use axios instead of fetch but we will revisit it after researching more